dev/com.ibm.ws.security.common/.classpath

```diff
@@ -4,7 +4,5 @@
 	<classpathentry kind="src" output="bin_test" path="test"/>
 	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
-	<classpathentry kind="src" path="/com.ibm.ws.webcontainer.security"/>
-	<classpathentry kind="src" path="/com.ibm.ws.security.test.common"/>
```

Comment:
I had forgotten that BND should be able to grab everything that's needed without updates to the `.classpath` file.